torch/_tensor_docs.py

```diff
@@ -1527,7 +1527,7 @@ def callable(a, b) -> number
 
 add_docstr_all('random_',
                r"""
-random_(from=0, to=None, *, generator=None) -> Tensor
+random_(min=0, max=None, *, generator=None) -> Tensor
```

Comment:
This makes me feel uneasy about backwards compatibility. For example,

`tensor.random_(to=10)` won't work anymore. Would renaming "from" to "start" and keeping "to" work?

The alternative is to somehow keep around the `to` keyword argument while having "min, max" args but that seems like more work than it is worth.